Ping q66 if this needs updating.

This prevents vscode from fetching prebuilt ripgrep from Microsoft
during build, which unbreaks build on platforms where MS deos not
ship a prebuilt ripgrep.

diff --git a/package.json b/package.json
index 6d1fb7cca0c..057ed39ac31 100644
--- a/package.json
+++ b/package.json
@@ -83,7 +83,7 @@
     "vscode-oniguruma": "1.6.1",
     "vscode-proxy-agent": "^0.11.0",
     "vscode-regexpp": "^3.1.0",
-    "vscode-ripgrep": "^1.12.1",
+    "vscode-ripgrep": "https://github.com/q66/vscode-ripgrep.git",
     "vscode-textmate": "5.5.0",
     "xterm": "4.16.0-beta.2",
     "xterm-addon-search": "0.9.0-beta.6",
diff --git a/remote/package.json b/remote/package.json
index 662ce3568ab..a79fe3764d9 100644
--- a/remote/package.json
+++ b/remote/package.json
@@ -22,7 +22,7 @@
     "vscode-oniguruma": "1.6.1",
     "vscode-proxy-agent": "^0.11.0",
     "vscode-regexpp": "^3.1.0",
-    "vscode-ripgrep": "^1.12.1",
+    "vscode-ripgrep": "https://github.com/q66/vscode-ripgrep.git",
     "vscode-textmate": "5.5.0",
     "xterm": "4.16.0-beta.2",
     "xterm-addon-search": "0.9.0-beta.6",
